XRTK-Core/Packages/com.xrtk.core/Inspectors/Utilities/SymbolicLinks/SymbolicLinker.cs

```diff
@@ -374,11 +374,33 @@ private static bool CreateSymbolicLink(string sourceAbsolutePath, string targetA
                 Directory.Delete(targetAbsolutePath);
             }
 
-            // --------------------> /C mklink /D "C:\Link To Folder" "C:\Users\Name\Original Folder"
-            if (!new Process().Run($"/C mklink /D \"{targetAbsolutePath}\" \"{sourceAbsolutePath}\"", out var error))
+            switch (Environment.OSVersion.Platform)
             {
-                Debug.LogError($"{error}");
-                return false;
+                case PlatformID.MacOSX:
+                    // --------------------> ln -s /path/to/original /path/to/symlink
+                    if (!new Process().Run($"ln -s \"{sourceAbsolutePath.ToForwardSlashes()}\" \"{targetAbsolutePath.ToForwardSlashes()}\"", out var error))
```

Comment:
Need to add `"/Applications/Utilities/Terminal.app/Contents/MacOS/Terminal"`

and the terminal doesn't seem to work with relative paths.